pkg/apis/pingcap/v1alpha1/validation/validation.go

```diff
@@ -116,6 +120,21 @@ func validatePDSpec(spec *v1alpha1.PDSpec, fldPath *field.Path) field.ErrorList
 	return allErrs
 }
 
+func validatePDAddresses(arrayOfAddresses []string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	for i, address := range arrayOfAddresses {
+		idxPath := fldPath.Index(i)
+		u, err := url.Parse(address)
+
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(idxPath, address, err.Error()))
```

Comment:
The error shows the correct address here right?
Would you mind give an example in the PR description?

Reply:
The PR description already includes an format example: `http://{address}:{port}`, and I've add example in error log.